### ISSUE TITLE
Add Android 12 Monet colors as sensors into Home Assistant

### ISSUE THREAD

**Issue by Golbinex** [home-assistant/android#2085]
**Is your feature request related to a problem? Please describe.**

Android 12 introduced dynamic UI accent colors based on a wallpaper (also called Material You). Colors are applied to every UI element which supports it. HA android app could make use of these colors as well and introduce them into Home Assistant system as sensors.
For example, light bulbs could be colored like wallpaper on the phone.
**Describe the solution you'd like**

Extract accent color codes from android and add them as sensors into Home Assistant. Only few color categories are needed, perhaps A1-200 (@android:color/system_accent1_200) is the most suitable one, as it seems to be the brightest main color of the wallpaper.
**Describe alternatives you've considered, if any**

Copying color code from the Android S Easter egg widget and pasting it into Home Assistant is also solution, however it's not automated and not viable if your wallpaper is regularly changing.
**Additional context**

**Comment by dshokouhi** [comment]
Started to look at this request and found the following stackoverflow post: https://stackoverflow.com/questions/71374425/how-to-get-material3-primary-accent-colors-programmatically

Do we have an idea of which color variables we need? So far I can see that `primary` `accent` and `tertiary` all become different values as I pick a different dynamic theme color. Are these what we would expect to send as a value? Also do we think a hex format will suffice? Maybe we can start with these 3 sensors and add more if users find more to add? 

Here is an example of the output I am seeing.

```
primary color is #1B1B1F, accent is #435E91, tertiary is #73767E
primary color is #191C1A, accent is #266A4A, tertiary is #6F7871
``` 

I have looked at `secondary` and `foreground` but they are the same value as `primary`
I have looked at `background` and despite a different hex value its the same off white color
